### PR TITLE
initial model for course

### PR DESCRIPTION
**Pull request by koushikkm**
This provides the initial model for course module.
In the initial module we assume that we will have text only course. We will later extend it to contain video etc.

### CODE REVIEW

**Comment by koushikkm** [comment]
@pankajssp

File: course/models.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   
+
+class Module(models.Model):
+    """
+    Each course is divided into several modules. A module may contain text, video, or graphics
+    """
+    title = models.CharField(max_length = 250, help_text = 'title of this module')
+    course = models.ForeignKey(Course, related_name = 'module', help_text = 'individual module of a course')
+    #order = TODO: need to define some oreder here, probably some sequence number
+    content = models.TextField(help_text = 'actual content, limiting to text only content now') #TODO: a serious limitation
```
Comment (pankajssp) [inline course/models.py]: You say that " Each course is divided into several modules. A module may contain text, video, or graphics" , then should this class act as an abstract class and individual module type(like text, video, graphics etc.) be subclasses of the 'Module' abstract class? And therefore this 'content' field moves to derived class(concrete class) pertaining to text based say 'TextModule' class.

**Comment by koushikkm** [comment]
@pankajssp : added support for admin for course.However to merge it , we need to merge the blog part first and then resolve the conflict.

**Comment by pankajssp** [comment]
Please take care of the comment rest all looks good.

**Comment by koushikkm** [comment]
Opening it to merge

**Pull request merged**